src/locales/en.json

```diff
@@ -17,7 +17,7 @@
     "welcome_tab_2_content_heading_1": "By email",
     "welcome_tab_2_content_1_block_1_1": "Send the ",
     "gar_form_link": "https://www.gov.uk/government/publications/general-aviation-operators-and-pilots-notification-of-flights",
-    "welcome_tab_2_content_1_block_1_2": "General Aviation Report form",
+    "welcome_tab_2_content_1_block_1_2": "General Aviation Report form (opens in a new window)",
```

Comment:
Accessibility articles online suggest that links that do things like open a new tab or window should have it noted for a screen reader to inform a user.